internal/layer2/announcer.go

```diff
@@ -30,8 +34,10 @@ func New(l log.Logger) (*Announce, error) {
 		ndps:     map[int]*ndpResponder{},
 		ips:      map[string]net.IP{},
 		ipRefcnt: map[string]int{},
+		spamCh:   make(chan net.IP, 1024),
```

Comment:
What happens if the channel is full? What is the implication for the user?

Reply:
If `spamCh` is full `SetBalancer` blocks until we are done spamming in `spamLoop` and empty `spamCh` again
I expect people to have less than 100 layer2 VIP per cluster so 1024 should be good. If I'm wrong full resync will block for a bit which is fine.